libs/documentation/src/lib/components/formly-search/demos/advanced-search/advanced-search.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+
+@Component({
+  selector: 'gsa-sam-advanced-search',
+  templateUrl: './advanced-search.component.html',
+  styleUrls: ['./advanced-search.component.scss']
+})
+export class AdvancedSearch {
+
+  form = new FormGroup({});
+  model: any = {advanceSearch: "Advanced Search"}; 
+  options: FormlyFormOptions = {};
+  advancedSearchUrl = "/documentation/components/formly-search/examples";
+ 
+  fields: FormlyFieldConfig[] = [
+    {
+      key: 'advanced search',
+      templateOptions: {
+        label: 'Keyword (with label)',
+        ariaHidden: true,
+      },
+      fieldGroup: [
+        {
+          key: 'ddsearchmodel',
+          type: 'search',
+          templateOptions: {
+            label: 'Search with dropdown',
```

Comment:
No need for label & description if they are going to be hidden with hideLabel & hideDescription

Reply:
I think we can leave the label, even when hiding the wrapper should add 508 sr-only attribute

Reply:
Dave is right, can ignore